eng/scripts/Swagger-Prettier-Check.ps1

```diff
@@ -11,8 +11,8 @@ $repoPath = Resolve-Path "$PSScriptRoot/../.."
 $pathsWithErrors = @()
 
 if ($CheckAll) {
-  LogInfo "npx --no -- prettier --check $repoPath/specification/**/*.json --log-level warn"
-  npx --no -- prettier --check $repoPath/specification/**/*.json --log-level warn
+  LogInfo "npm exec --no -- prettier --check $repoPath/specification/**/*.json --log-level warn"
```

Comment:
I'm fine with this change but I'm curious about one question if you do use `npx --no -- ` do you still risk hitting the arg parsing issue?

Reply:
I **think** `npx --no --` should be identical (or very nearly identical) to `npm exec --no --`.  I think the differences are only when you **don't** use `--`.  But I couldn't be completely sure from the docs:

https://docs.npmjs.com/cli/v10/commands/npx#npx-vs-npm-exec

Either way, I think we should try to use `npm exec` everywhere in scripts going forward, and convert existing code where it makes sense.  If an existing script is using `npx` without issues, no hurry to change.